controllers/vpcendpoint/helpers.go

```diff
@@ -356,7 +356,7 @@ func (r *VpcEndpointReconciler) generateMissingSecurityGroupRules(ctx context.Co
 	}
 
 	if len(sourceSgIds) == 0 {
-		return nil, nil, fmt.Errorf("failed to find source security groups by default tags")
+		r.log.V(1).Info("Unable to find source security groups")
```

Comment:
Can we change this to an Error log? Unfortunately Info logs are not showing up in DT.

Reply:
Resolved in https://github.com/openshift/aws-vpce-operator/commit/754893fa1794bb4e6a86f379919bc78c17a3eb52